src/ui/settings/ViewGiveFeedback.qml

```diff
@@ -334,6 +340,15 @@ Item {
                    Layout.fillWidth: true
                    Layout.preferredHeight: Theme.rowHeight
                    onClicked: stackview.pop(StackView.Immediate)
+                   anchors.horizontalCenter: parent.horizontalCenter
+                   Component.onCompleted: {
+                    if (window.fullscreenRequired()) {
+                        anchors.bottom = parent.bottom;
```

Comment:
Thank you @mbirghan! A few last little things that I should have mentioned (sorry!) : Because this VPNButton {} is child of a ColumnLayout {}, we need to use Layout properties, instead of anchors, to position it (more on that [here](https://doc.qt.io/qt-5/qtquicklayouts-overview.html)) which would like this:

```
Component.onCompleted: {
  if (windows.fullscreenRequired()) {
    Layout.alignment = Qt.AlignBottom;
    Layout.bottomMargin = Theme.windowMargin * 4;
  }
}


The anchors in L320-325 are fine though since ColumnLayout {} is a child of Item {} and not another Layout.